src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/SummaryReportTask.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.slf4j.Logger;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.blobrouter.reconciliation.service.SummaryReportService;
+
+import java.time.LocalDate;
+
+import static java.time.ZoneOffset.UTC;
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.hmcts.reform.blobrouter.util.TimeZones.EUROPE_LONDON;
+
+@Component
+@ConditionalOnProperty(value = "scheduling.task.create-summary-report.enabled")
+public class SummaryReportTask {
```

Comment:
it's worth clarifying it's about reconciliation. Either in class name or package